<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QWidget" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:qlineargradient(spread:pad, x1:0.409, y1:0.420455, x2:1, y2:1, stop:0 rgba(143, 0, 255, 255), stop:1 rgba(255, 255, 255, 255))</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_id">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(38, 38, 38);</string>
   </property>
   <property name="placeholderText">
    <string>  Enter student ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_first">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(38, 38, 38);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>  Enter student First Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_last">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(38, 38, 38);</string>
   </property>
   <property name="placeholderText">
    <string>  Enter student Last Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(61, 41, 128);</string>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_email">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(38, 38, 38);</string>
   </property>
   <property name="placeholderText">
    <string>  Enter student Email</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_modify">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(61, 41, 128);</string>
   </property>
   <property name="text">
    <string>MODIFY</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delet">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>200</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(61, 41, 128);</string>
   </property>
   <property name="text">
    <string>DELETE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>  STUDENTS  INFORMATION</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>661</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(38, 38, 38);</string>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>FIRST NAME</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>LAST NAME</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>EMAIL</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton_list">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>229</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(61, 41, 128);</string>
   </property>
   <property name="text">
    <string>LIST</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
